tools/test-static-code

```diff
@@ -63,8 +63,10 @@ fi
 # content-security-policy entries. This is the browser equivalent
 # of setenforce 0
 #
+# We allow "img-src *" since that seems to be fairly harmless and is
+# needed in pkg/apps for screenshots.
```

Comment:
I wouldn't allow that in all packages generally, as this is a common technique for tracking users. So maybe make the grep specific to apps?

Reply:
Yes, fixed.